### PR TITLE
Component unification (Register Trees)

### PR DESCRIPTION
**Pull request by prachigarg19**
Unify mui components in the webapp.

### CODE REVIEW

**Comment by mariahosfeld** [comment]
Functionality looks good to me.

File: src/features/user/RegisterTrees/RegisterTreesForm.tsx [hunk 1]
```diff
@@ -49,15 +49,7 @@ const dialogSx: SxProps = {
   },
 };
 
-type FormData = {
-  treeCount: string;
-  species: string;
-  plantProject: string | null;
-  plantDate: Date;
-  geometry: any;
-};
```
Comment (sunilsabatp) [inline src/features/user/RegisterTrees/RegisterTreesForm.tsx:58]: was there any reason that you removed type  formData ?

Comment (prachigarg19) [reply src/features/user/RegisterTrees/RegisterTreesForm.tsx:58]: It must've been removed while merging. I will add it again.

**Review by sunilsabatp — APPROVED** [review]
LGTM

**Pull request merged**